sample_files/.travis.yml

```diff
@@ -14,8 +14,10 @@ env:
   - VERSION="8.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
   - TRANSIFEX_USER='transbot@odoo-community.org'
   # This line contains the encrypted transifex password
-  # To encrypt transifex password, use
-  # $ travis encrypt TRANSIFEX_PASSWORD=your-password -r owner/project
+  # To encrypt transifex password, install travis ruby utils with:
+  #   $ [sudo] gem install travis
```

Comment:
Best doc I found is here[1]
If you add `--user-install` it will install it in `/home/username/.gem/ruby/1.9.1/bin`

[1] http://www.binarytides.com/quick-tip-installing-ruby-gems-in-the-users-home-directory/